salt/modules/influxdbmod.py

```diff
@@ -442,16 +455,17 @@ def create_retention_policy(database,
     '''
     client = _client(**client_args)
     client.create_retention_policy(name, duration, replication, database,
-                                   default)
+                                   default, shard_duration)
 
     return True
 
 
 def alter_retention_policy(database,
                            name,
-                           duration,
-                           replication,
-                           default=False,
+                           duration=None,
+                           replication=None,
+                           default=None,
```

Comment:
Can you elaborate as to why having `default=False` is causing issues? I did notice on the api documentation that the default value should be None but just trying to understand the issue this is fixing

Reply:
@Ch3LL You're right, after a few more tests, default=False is the same as default=None. I thought it had fixed my issue probably because of a side effect: the "default" value was already changed in my influxdb database.